src/storage-manager/deploy/storage-manager.yaml.template

```diff
@@ -68,6 +68,8 @@ spec:
         - name: host-modules
           mountPath: /lib/modules
```

Comment:
what is `/ilb/modules` used for?

Reply:
If think it used for:
https://github.com/microsoft/pai/blob/120179950934692bd304ce5f38a54f0ecba673eb/src/storage-manager/deploy/scripts/entrypoint.sh#L46-L47

Reply:
Refer to https://stackoverflow.com/questions/33013539/docker-loading-kernel-modules